.env.example

```
@@ -16,14 +16,42 @@ S3_BUCKET=
+
+# ===============================
+# GOOGLE Login
+# ===============================
+GOOGLE_CLIENT_ID=
+GOOGLE_CLIENT_SECRET=
+GOOGLE_REDIRECT_URI=
+
+# ===============================
+# JWT Secret
+# ===============================
+JWT_SECRET=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)